### DISCUSSION TITLE
Different order of mods displayed on the leaderboards between stable and lazer scores of the same mod combination

### DISCUSSION THREAD

**Discussion by ikkue** [ppy/osu#18483] [Q&A] [answered]
The scores were set on [this map](https://osu.ppy.sh/beatmapsets/1077335#osu/2254470).
![osu_2022-05-30_00-41-07](https://user-images.githubusercontent.com/11213135/170884136-6846de0a-1fce-4405-8e32-7fb2e294f710.png)

As you can see, the score imported from [stable](https://drive.google.com/file/d/1oCrxL30a0Trfj0LCfFaxEvZjNaj3YFO0/view?usp=sharing) is ordered as HD RX CL.
![osu_2022-05-30_00-41-13](https://user-images.githubusercontent.com/11213135/170884139-6f1ae547-d5d5-447b-9b3f-5b2782baa33c.png)

But the score set in [lazer](https://drive.google.com/file/d/1hQPwibd2lYz7jo8A3zT3qbPE_kRZcc-n/view?usp=sharing) with the same mod combination is ordered as HD CL RX.
![osu_2022-05-30_00-41-15](https://user-images.githubusercontent.com/11213135/170884135-9b7227df-e457-404d-a07f-68415a0b51b0.png)

I don't know if this is a known "issue", or if it's even an issue at all, but even if it's a minute issue, it's still worth pointing out.

**Comment by bdach** [comment, marked as answer]
we'll probably decide on a canonical ordering and apply it uniformly at some point. super low priority though.

**Reply by ikkue** [reply]
I guessed that it's gonna be a low priority, but it's nice to get a proper answer on records anyways.

Also, I'm guessing that the reason why it's like that is probably because the game always puts the Classic mod last for scores set in / imported from stable.

In my opinion, if that was done deliberately to differentiate between stable and lazer scores, there should be a separate tag for scores set in / imported from stable instead.

It could be like the "Open", "Spotlight", or "Featured Artist" tag on Playlists but right beside the usernames and in purple saying "Classic".

**Reply by bdach** [reply]
the classic mod will remain a mod and not a "tag". it's the accepted direction forward.

**Reply by ikkue** [reply]
What I was implying is that the Classic mod stays as is, but scores that are set in / imported from stable should have an additional tag differentiating them from scores set in lazer. Basically, the tag would indicate that the score was set in / imported from the "Classic Client" AKA what's currently called "stable", as "lazer" now will eventually become the new "stable" as I understand it.

TL;DR Classic mod stays the same, but an additional "Classic" tag on the score helps differentiate scores that were set on the "classic client" AKA "stable"

**Reply by bdach** [reply]
why? at that point you have two indicators that indicate mostly the same thing. what's the point?

**Reply by Walavouchey** [reply]
I recommend you take a look at [community meeting 12](https://www.youtube.com/watch?v=HimCHAnPCCY), which discusses how the Classic mod will work in relation to stable and lazer, if you haven't seen it already.

As for differentiating between stable scores and Classic mod scores, as viewed in lazer (they'd both have the Classic mod applied), the point of the Classic mod is to completely emulate stable gameplay, so it wouldn't matter which client version they're from even if they're displayed on the same leaderboard.
